lib/tasks/annotations.rake

```diff
@@ -0,0 +1,25 @@
+namespace :annotations do
+  desc 'Compare Ruby\'s text to what Vue returns'
+  task(:compare_ruby_text_to_vue => :environment) do
+    [Case, TextBlock].each do |klass|
+      i = 0
+      count = klass.count
+      klass.find_each do |inst|
+        i += 1
+        ruby_text = HTMLFormatter.parse(inst.content).text
+        vue_text = HTMLFormatter.parse(Vue::SSR.render(inst.content)).text
```

Comment:
I think this is supposed to be `HTMLUtils`

Reply:
Good catch!

Reply:
Fixed in 9807bd3a3e791783fe2f2469b52000a41eeffc49